examples/da/plot_otda_classes.py

```diff
@@ -15,8 +15,10 @@
 # License: MIT License
 
 import matplotlib.pylab as pl
+import numpy as np
 import ot
 
+np.random.seed(42)
```

Comment:
Actually I think @agramfort  prefer to use RandomSate as explained in
https://stackoverflow.com/questions/22994423/difference-between-np-random-seed-and-np-random-randomstate

This allow you to use controlable random generator without impacting the global one as is done in random.seed.

Note that we plan on doing proper dataset functions that accept a random state generator as parameter to allow full control (See #21 ).

Reply:
no worries ;)